.travis.yml

```diff
@@ -3,6 +3,15 @@ language: java
 services:
   - redis-server
 
+before_install:
```

Comment:
Can we just use services as shown https://docs.travis-ci.com/user/database-setup/#CouchDB